src/vfxt/src/createUiDefinition.json

```diff
@@ -151,10 +151,10 @@
               "name": "adminPassword",
               "type": "Microsoft.Common.PasswordBox",
               "label": {
-                "password": "vFXT Node Password",
+                "password": "Cluster administration password",
                 "confirmPassword": "Confirm password"
               },
-              "toolTip": "The admin password used for the Avere vFXT cluster.",
+              "toolTip": "The password used with the username 'admin' to configure the Avere vFXT cluster.",
```

Comment:
This is also the 'root' user password, I believe. I don't know if we need to call this out, but if we did:
The password used with the 'admin' and 'root' users of the Avere vFXT cluster.